src/stories/Form/component.stories.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+import StyledForm from "./index";
+
+export default {
```

Comment:
create another const props  for your form input.

Reply:
done